e2e/actions/l2_challenger_test.go

```diff
@@ -66,7 +66,99 @@ interaction:
 	require.Equal(rt.t, chal.StatusApproved, status)
 }
 
-func TestChallengerProvingTimeout(t *testing.T) {
+func TestChallengeAsserterBisectTimeout(t *testing.T) {
+	rt := defaultRuntime(t)
+
+	rt.setTargetInvalidBlockNumber(testdata.TargetBlockNumber)
+	rt.setupMaliciousValidator()
+	rt.setupHonestChallenger()
+	rt.setupHonestGuardian()
+
+	// bind contracts
+	rt.bindChallengeContracts()
+
+	// create challenge
+	rt.setupChallenge()
+
+interaction:
+	for {
+		status, err := rt.colosseumContract.GetStatus(nil, rt.outputIndex)
+		require.NoError(rt.t, err)
+
+		switch status {
+		case chal.StatusAsserterTurn:
+			// do nothing to trigger asserter timeout
+			rt.miner.ActEmptyBlock(rt.t)
+		case chal.StatusAsserterTimeout, chal.StatusReadyToProve:
```

Comment:
In this test case, `StatusReadyToProve` should not come. If `StatusReadyToProve` comes by a bug, this test case should be failed.
```suggestion
		case chal.StatusAsserterTimeout:
```